blog/_posts/2019-01-18-fluxdiffeq.md

```diff
@@ -0,0 +1,672 @@
+[DiffEqFlux.jl](https://github.com/JuliaDiffEq/DiffEqFlux.jl/),
+we will show the reader how to easily add differential equation
+layers to neural networks using a range of differential equations models, including stiff ordinary
+differential equations, stochastic differential equations, delay differential
+equations, and hybrid (discontinuous) differential equations.
+
+This is the first user-friendly toolbox to combine a fully-featured differential
+equations solver library and neural networks seamlessly together. The blog post will also show
+why the flexibility of a full differential equation solver suite is necessary
+for doing this properly. With the ability to fuse neural networks
+seamlessly with ODEs, SDEs, DAEs, DDEs, stiff equations, and different methods
+for adjoint sensitivity calculations, this is a large generalization of the
+neural ODEs work and will allow researchers to better explore the problem
+domain.
+
+(Note: If you are interested in this work and are an undergraduate or graduate
+student, we have [Google Summer of Code projects available in this area](https://julialang.org/soc). This
+[pays quite well over the summer](https://developers.google.com/open-source/gsoc/help/student-stipends).
+Please join the [Julia Slack](https://slackinvite.julialang.org/) and the #jsoc channel to discuss in more detail.)
+
+## What do differential equations have to do with machine learning?
+
+The first question someone not familiar with the field might ask is, why are
+differential equations important in this context? The simple answer is that a
+differential equation is a way to specify an arbitrary nonlinear transform by
+mathematically encoding prior structural assumptions.
+
+Let's unpack that statement a bit. There are generally three ways
+to define a nonlinear transformation. The first is to explicitly say what it is.
+For example, the sigmoid function is $\sigma(x)=\frac{e^x}{e^x + 1}$. This only works
+if you know the exact functional form that relates the input to the output.
+However, in many cases, such exact relations are not known *a priori*.
+So how do you do nonlinear modeling if you don't know the nonlinearity?
+
+There are two separate ways to then handle this problem. One method is machine
+learning. Usually this is presented as you have some $x$ and you want to predict
+$y$. This generation of a prediction from $x$ is the machine learning model, call it
+$ML$, where you put $x$ into the program and it spits out values $y$. But if you think
+about it differently, this is actually just a nonlinear transformation $y=ML(x)$.
+The reason why the model $ML$ is interesting is because its form is basic but adapts to the
+data itself. For example, a simple neural network (in design matrix form) with
+sigmoid activation functions is simply matrix multiplications followed
+by sigmoids, meaning $ML(x)=\sigma(W_{3}\cdot\sigma(W_{2}\cdot\sigma(W_{1}\cdot x)))$ is a three-layer deep

```

Comment:
`by sigmoids, meaning` => `by application of a sigmoid function.  Specifically, `